include/darshan-log-format.h

```diff
@@ -163,7 +163,8 @@ struct darshan_base_record
     X(DARSHAN_MPIIO_MOD,    "MPI-IO",     DARSHAN_MPIIO_VER,     &mpiio_logutils) \
     X(DARSHAN_H5F_MOD,      "H5F",        DARSHAN_H5F_VER,       &hdf5_file_logutils) \
     X(DARSHAN_H5D_MOD,      "H5D",        DARSHAN_H5D_VER,       &hdf5_dataset_logutils) \
-    X(DARSHAN_PNETCDF_MOD,  "PNETCDF",    DARSHAN_PNETCDF_VER,   &pnetcdf_logutils) \
+    X(DARSHAN_PNETCDF_FILE_MOD, "PNETCDF_FILE",  DARSHAN_PNETCDF_FILE_VER, &pnetcdf_file_logutils) \
+    X(DARSHAN_PNETCDF_VAR_MOD,  "PNETCDF_VAR",   DARSHAN_PNETCDF_VAR_VER,  &pnetcdf_var_logutils) \
```

Comment:
Does this mean that another module id number was inserted in the middle of the list, which would make higher number ones (like stdio) get new numbers)?  I haven't checked out the code and looked carefully at it yet but I wanted to mention this since it is potentially a big deal.

Reply:
I feel like this came up in previous discussions I had with Shane if I'm not mistaken. It did cause me to make a bunch of adjustments IIRC. I know that for libraries with substantial consumers of their API appending is usually most desirable, even if you leave an old/dead entry behind.

Reply:
Ah yeah, you can see the various off-by-1 error discussions above I think -- here is 1 of them: https://github.com/darshan-hpc/darshan/pull/456#discussion_r964187665

Reply:
That's correct. There's some logic in darshan-utils that handles the module shift in the log header.

I agree not a great idea in hindsight. I did the same thing for the original HDF5 module split, so basically just copied the strategy here, for better or worse. I'll make sure not to make a habit out of it.

I would think it's not really a concern for end users though -- there's not any reason for them to be using specific integer IDs for modules (technically they can't even be sure an ID is an integer), rather than the named enums, and logutils makes sure to account for the shift.

Reply:
Ahh, I see.  That's the memmove stuff that shifts the mod_map and mod_ver stuff around for older logs.  Ok!  That's fine as long as its accounted for.